### ISSUE TITLE
Incorrect fix for `import-x/no-duplicates`

### ISSUE THREAD

**Issue by ocavue** [un-ts/eslint-plugin-import-x#449]
Reproduction: https://github.com/issueset/eslint-plugin-import-x-svelte-issue.git

```bash
$ git clone https://github.com/issueset/eslint-plugin-import-x-svelte-issue.git

$ cd eslint-plugin-import-x-svelte-issue

$ pnpm install

$ cat ./eslint.config.mjs
import js from "@eslint/js";
import { defineConfig } from "eslint/config";
import * as tseslint from 'typescript-eslint';
import { importX } from 'eslint-plugin-import-x';
const config = defineConfig([
  js.configs.recommended,
  tseslint.configs.recommended,
  importX.flatConfigs.recommended,
  importX.flatConfigs.typescript,
  {
    files: ['**/*.{js,mjs,cjs,jsx,mjsx,ts,tsx,mtsx}'],
    settings: {
      'import-x/resolver': {
        typescript: true
      }
    },
    rules: {
      'import-x/no-duplicates': ['warn', { 'prefer-inline': true }]
    },
  },
])
export default config

$ cat ./src/index.ts
import { onMount } from 'svelte'
import { readonly, writable, type Readable } from 'svelte/store'
export function f(r: Readable<number>) {
    return `${r} ${readonly} ${writable} ${onMount}`
}

$ pnpm run fix
> eslint --fix .

$ cat ./src/index.ts
import { onMount, readonly, writable, type Readable  } from 'svelte'
export function f(r: Readable<number>) {
    return `${r} ${readonly} ${writable} ${onMount}`
}

$
```

Notice that the `svelte/store` from `src/index.ts` was incorrectly removed after running `eslint --fix`.

**Comment by nnmrts** [comment]
Same issue with these imports:

```js
import { print } from "esrap";
import ts from 'esrap/languages/ts';
```

**Comment by rogerfar** [comment]
Happens with svelte too:

```
	import { onDestroy, onMount } from 'svelte';
	import { quartIn, quartInOut } from 'svelte/easing';
	import { draw, fade } from 'svelte/transition';
```
After:
```
	import { onDestroy, onMount, quartIn, quartInOut , draw, fade  } from 'svelte';
```

That's wrong.
